### DISCUSSION TITLE
Created ID from a Save Row

### DISCUSSION THREAD

**Discussion by jypogoy** [Budibase/budibase#6763] [Help] [answered]
Hi Budibase Team! Would there be any way that I can get the created record's ID from a Save Row action? This way, I can use it on another Save Row for a related table in MySQL or in any database. This same goes also with other query executions.

**Comment by melohagan** [comment, marked as answer]
Hey @jypogoy 

You could have a look at this: https://docs.budibase.com/docs/referencing

**Reply by jypogoy** [reply]
Overlooked that portion. Though with some parsing on the extra %5B and %5D. Thanks a lot!
